rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -1202,21 +1202,8 @@ def test_upgrade_db_35_to_36(user_data_dir):  # pylint: disable=unused-argument
             else:
                 assert entry[i] == tx_address_mappings_result[idx][i - 1]
     new_tx_mappings_result = cursor.execute('SELECT * from evm_tx_mappings').fetchall()  # noqa: E501
-    assert len(new_tx_mappings_result) == 305
-    for idx, entry in enumerate(new_tx_mappings_result):
-        for i in range(3):
-            if i == 0:
-                assert entry[i] == tx_mappings_result[idx][i]
-            elif i == 1:
-                assert entry[i] == 1
-            else:
-                if entry[i] == 0:
-                    assert tx_mappings_result[idx][i] == 'decoded'
-                elif entry[i] == 1:
-                    assert tx_mappings_result[idx][i] == 'customized'
-                else:
-                    raise AssertionError(f'Unexpected value {entry[i]} in evm_tx_mappings')
-    assert cursor.execute('SELECT COUNT(*) from history_events').fetchone()[0] == 562
```

Comment:
all the history events are evm_transactions? If there was a customized event there should be at least one that is not right?